### PR TITLE
Wasm lazy loading sample

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

File: common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Pages/Counter.razor [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿@page "/counter"
```
Comment (Stamo-Gochev) [inline common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Pages/Counter.razor:1]: I think this page can be removed - this put the focus on 2 options only - lazy loaded page and non-lazy-loaded page.

File: common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Shared/CultureChooser.razor [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    </select>
+</div>
+
+@code{
+    private protected readonly CultureInfo[] SupportedCultures = new[]
+       {
+            new CultureInfo("en-US"),
+            new CultureInfo("bg-BG")
+        };
+
```
Comment (Stamo-Gochev) [inline common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Shared/CultureChooser.razor:23]: The indentation can be fixed.

File: common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Shared/TelerikLayout.razor [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿@inherits LayoutComponentBase
+
+@* Contains the Telerik Root Component, because it is a component coming from the Telerik assembly, so you can't have that for all pages if you want to load the Telerik assembly on demand. Due to the limitations of the framework with regards to registering services, this is where we now provie the localization service. *@
```
Comment (Stamo-Gochev) [inline common/lazy-load-assemblies-wasm/LazyLoadTelerikComponents/Client/Shared/TelerikLayout.razor:3]: The comment can be split into multiple lines, so you can read it more easily.

File: common/lazy-load-assemblies-wasm/readme.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+This sample project demonstrates how you can use the <a href="https://docs.microsoft.com/en-us/aspnet/core/blazor/webassembly-lazy-load-assemblies?view=aspnetcore-5.0" target="_blank">lazy assembly loading feature of Blazor</a> with the Telerik components.
+
+## Requirements
+
+There are a few key points and changes to a standard project with relation to the Telerik components:
+
+1. Move the `<TelerikRootComponent>` to a layout that is used only on pages that have the Telerik assemblies loaded.
+
+1. Mark the required assemblies as lazy-loaded in the `.csproj` file of the web assembly project - the following snippet shows the assemblies that the Telerik components rely on at the time of the `2.19.0` release.
```
Comment (Stamo-Gochev) [inline common/lazy-load-assemblies-wasm/readme.md:11]: We can make it clear that not all of these assemblies need to be lazy loaded (referring to the part "Mark the required assemblies") - it is up to the developer to decide, but they can choose a subset of that. For example, `System.Linq.Queryable` is a "common" package that might have been already loaded.

**Review by Stamo-Gochev — APPROVED** [review]

**Comment by Stamo-Gochev** [comment]
Not sure about the convention here, but I think that squashing the commits results in an easier tracking of the change.

**Comment by marin-bratanov** [comment]
The merge strategy will definitely be squash and merge. Pending the official .NET5 release for the package versions, and a review of the documentation and comments part by @svdimitr  and I will merge it.

**Review by svdimitr — APPROVED** [review]

**Pull request merged**